tools/github-issues/Azure.Sdk.Tools.GitHubIssues/Reports/CustomerReportedInvalidState/FindCustomerRelatedIssuesInvalidState.cs

```diff
@@ -107,7 +107,9 @@ private bool ValidateIssueType(Issue issue, StringBuilder problemsWithTheIssue)
 
         private bool ValidateIssueAssignee(Issue issue, StringBuilder problemsWithTheIssue)
         {
-            if (issue.Assignee == null)
+            // if the issue is not assigned to anyone AND
+            // the issue is NOT in the backlog milestone (a milestone with no dueOn date)
+            if (issue.Assignee == null && !(issue.Milestone.DueOn != null))
```

Comment:
Could this be simplified to:

`if (issue.Assignee == null && issue.Milestone.DueOn == null)`?   Or is there some weirdness to watch out for.

Reply:
Issues not assigned to a person and NOT in a backlog milestone should be flagged
- `issue.Assignee == null;` // Does not have an owner
- `issue.Milestone.DueOn != null` // Is NOT a backlog milestone

I think this is right... It took me way too long to parse this :) -- I pushed a change.